playbooks/besman-sonar-scanner-0.0.1-playbook.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+# Author: Samir Ranjan Parhi
+# License: Same as Repository Licence
+# Usage : Beta-Release
+# Date : 12/02/2024
+
+function __besman_init() {
+    __besman_echo_white "initialising"
+    export ASSESSMENT_TOOL_NAME="sonar-scanner"
+    export ASSESSMENT_TOOL_TYPE="sonar-scanner"
+    export ASSESSMENT_TOOL_VERSION="v0.0.1"
```

Comment:
Is the version of sonar-scanner used v0.0.1?

Reply:
is this verison of the tool or version of the variation of this code .

Reply:
```
local var_array=("BESMAN_ARTIFACT_TYPE" "BESMAN_ARTIFACT_NAME" "BESMAN_ARTIFACT_VERSION" "BESMAN_ARTIFACT_URL" "BESMAN_ENV_NAME" "BESMAN_ARTIFACT_DIR" "ASSESSMENT_TOOL_NAME" "ASSESSMENT_TOOL_TYPE" "ASSESSMENT_TOOL_VERSION" "ASSESSMENT_TOOL_PLAYBOOK" "BESMAN_ASSESSMENT_DATASTORE_DIR" "BESMAN_TOOL_PATH" "BESMAN_ASSESSMENT_DATASTORE_URL" "BESMAN_LAB_OWNER_TYPE" "BESMAN_LAB_OWNER_NAME" "SONAR_HOST_URL" "SONAR_LOGIN_TOKEN")
```
is this the place we should keep it